packages/dds/map/src/map.ts

```diff
@@ -378,11 +378,5 @@ export class SharedMap extends SharedObject<ISharedMapEvents> implements IShared
      * {@inheritDoc @fluidframework/shared-object-base#SharedObject.registerCore}
      * @internal
      */
-    protected registerCore() {
-        for (const value of this.values()) {
```

Comment:
Have not spent any time looking deeper here, but do we have proper binding of handles in workflows where it's required? I.e. 
- if a handle is stored in attached map, it gets attached properly
- if map contains handles and map gets attached, it attached all handles stored in a map.

I'd think we do (otherwise existing code would not work correctly), but worth double checking, as that code did exactly that (with the structure we no longer support).

And I think (glancing quickly through the code) that registerCore() can be deleted?
I see that matrix and sharedstirng use it, but I'm not sure if usage is correct - I think this method is called when DDS binds to data store, but it means pretty much nothing (as DDS might still be not attached to a file?)

@agarwal-navin, @anthony-murphy, any chance you can glance to help Scarlet navigate it?

Reply:
I'm not sure I understand the first part of this comment correctly. I think attaching handles is unaffected by this change but we want to check that they do attach properly in order to definitively remove all support for putting SharedObjects inside SharedObjects?

Reply:
Yes, your code does not change behavior of handles. But delete code had same intention (of ensuring that objects are properly bound to map if referenced by map). I think handles are working properly but given that workflows are roughly the same for old way of doing things and when using handles, it's a bit surprising to see no code dealing with handles here.
Probably it's fine, but given that Navin recently looked at handle-related code, I though it's worth double checking that handle flows are Ok.

Reply:
there is also the possibility that there are bugs in the handle flow that get covered up by this flow

Reply:
Thanks Tony - yes, that's what I wanted to say (not sure why it took me too long to formulate it :))

Reply:
I took at a look at the DDSs I've modified here and they all have handling for handles (including cell). I created an issue to add unit test coverage: #8967.